### PR TITLE
[Nokia-7220 IXR] Add support for NOKIA 7220 IXR-H3, H4-32D, H5-64D platforms 

### PR DESCRIPTION
**Pull request by fzhou-nokia**
#### Why I did it
Add support for NOKIA 7220 IXR-H3, NOKIA 7220-IXR-H4-32D & 7220-IXR-H5-64D platforms:

>  **NOKIA 7220-IXR-H3:**
>     Platform: x86_64-nokia_ixr7220_h3-r0
>     HwSKU: Nokia-IXR7220-H3
>     ASIC: Broadcom
>     Port Config: 32x400G + 2x10G

>  **NOKIA 7220-IXR-H4-32D:**
> Platform: x86_64-nokia_ixr7220_h4_32d-r0
> HwSKU: Nokia-IXR7220-H4-32D
> ASIC: Broadcom
> Port Config: 32x400G + 1x10G

>  **NOKIA 7220-IXR-H5-64D:**
> Platform: x86_64-nokia_ixr7220_h5_64d-r0
> HwSKU: Nokia-IXR7220-H5-64D
> ASIC: Broadcom
> Port Config: 64x400G + 2x10G

##### Work item tracking
- Microsoft ADO **(number only)**:

#### How I did it
1.	Added module for each platform under ../device/nokia directory.
2.	Modified the platform/broadcom/one-image.mk to include each PLATFORM_MODULE
3.	Modified the platform/broadcom/ platform-modules-nokia.mk to include each platform
4.	Add platform related directory ixr7220h3, ixr7220h4-32d and ixr7220h5-64d in platform/Broadcom/sonic-platform-modules-nokia
5.	Modified related build scripts in platform/Broadcom/sonic-platform-modules-nokia/debian

#### How to verify it
1.	Make sure the sonic-buildimage is successful 
2.	Run this image on x86_64-nokia_ixr7220_h3-r0 and verify all dockers are up and test basic commands like:
                  o	show version
                  o	show platform summary
                  o	show platform syseeprom
                  o	show platform fan
                  o	show platform psustatus
                  o	show platform firmware status
                  o	show platform temperature
                  o	sudo show system-health detail
                  o	show interface status
3.	Run OC test, for T0/T1 topology and have 100% pass on platform.
4.	Run this image on x86_64-nokia_ixr7250e_sup-r0 and x86_64-nokia_ixr7250e_36x400g-r0, make sure all functions run correctly.

#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [ ] 202012
- [ ] 202106
- [ ] 202111
- [ ] 202205
- [ ] 202211
- [ ] 202305

#### Tested branch (Please provide the tested image version)

- [ ] 
- [ ]

### CODE REVIEW

**Comment by fzhou-nokia** [comment]
@Pavan-Nokia @jon-nokia @mlok-nokia Please help review

File: device/nokia/x86_64-nokia_ixr7220_h3-r0/plugins/led_control.py [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+                    oldfan = 0xf
+            else:
+                if (os.path.isfile(SWPLD1_DIR+"led_fan")):
+                    if oldfan != 0x0:
+                        self._write_sysfs_file(SWPLD1_DIR+"led_fan", "2")
+                        oldfan = 0x0
+                else:
+                    oldfan = 0xf
+
+            """ # Front Panel PSU1 Panel LED setting
```
Comment (jon-nokia) [inline device/nokia/x86_64-nokia_ixr7220_h3-r0/plugins/led_control.py:109]: In general, we should not commit untested and/or commented out code.  Save this for some other commit.

Comment (fzhou-nokia) [reply device/nokia/x86_64-nokia_ixr7220_h3-r0/plugins/led_control.py]: Thanks! I update it with [661c78b](https://github.com/sonic-net/sonic-buildimage/pull/18497/commits/661c78b26e0395eea6e72d6ee2c90092048f6765)

**Comment by Pavan-Nokia** [comment]
> > @liushilongbuaa @mssonicbld How can we help resolve this "internal conflict" issue and get this PR merged.
> 
> It is resolved last week.

I still see that the corresponding PR check is in pending state and has not completed.
Do you recommend we restart the checks?

**Comment by Blueve** [comment]
> Run this image on x86_64-nokia_ixr7250e_sup-r0 and x86_64-nokia_ixr7250e_36x400g-r0, make sure all functions run correctly.

Why you state that you tested on 7250?

**Comment by Pavan-Nokia** [comment]
@Blueve @yxieca Please help merge

**Comment by yxieca** [comment]
@prgeor can you help review this PR?

**Comment by Pavan-Nokia** [comment]
@liushilongbuaa After we rebased the PR to latest Master the internal conflict (ms_conflict) has appeared again.
Could you please help resolve.

**Review by prgeor — APPROVED** [review]

**Comment by liushilongbuaa** [comment]
> @liushilongbuaa After we rebased the PR to latest Master the internal conflict (ms_conflict) has appeared again. Could you please help resolve.

We are fixing the conflict. It still needs about 12 hours for the automation.

**Pull request merged**